### ISSUE TITLE
Type "Type[ArgumentDefaultsHelpFormatter]" cannot be assigned to type "_FormatterClass" (reportGeneralTypeIssues)

### ISSUE THREAD

**Issue by tekumara** [microsoft/pyright#846]
test.py:
```python
import argparse

def arg_parser():
    parser = argparse.ArgumentParser(
        description='myapp', formatter_class=argparse.ArgumentDefaultsHelpFormatter
    )
```

Produces:
```
  5:46 - error: Argument of type "Type[ArgumentDefaultsHelpFormatter]" cannot be assigned to parameter "formatter_class" of type "_FormatterClass" in function "__init__"
  Type "Type[ArgumentDefaultsHelpFormatter]" cannot be assigned to type "_FormatterClass" (reportGeneralTypeIssues)
```

pyright 1.1.52
Python 3.6.10

**Comment by erictraut** [comment]
Thanks for the bug report and the clear repro case. This will be fixed in the next version of Pyright and Pylance.

**Comment by erictraut** [comment]
This is now fixed in Pyright 1.1.54, which I just published. It will also be included in the next release of Pylance.

**Comment by tekumara** [comment]
Thank you @erictraut! 🙇 I've confirmed it's fixed in Pyright 1.1.54
